Documentation/rust/quick-start.rst

```diff
@@ -191,8 +191,12 @@ Configuration
 
 ``Rust support`` (``CONFIG_RUST``) needs to be enabled in the ``General setup``
 menu. The option is only shown if a suitable Rust toolchain is found (see
-above), as long as the other requirements are met. In turn, this will make
-visible the rest of options that depend on Rust.
+above), and the following other configurations are NOT set:
+
+- ``CONFIG_MODVERSIONS``
+- ``DEBUG_INFO_BTF``
```

Comment:
I'm not sure this list is exhaustive, and if this won't stay up to date, maybe we should just be more clear that `make rustavailable` exiting cleanly isn't the only prerequisite?

Reply:
Yeah, that is what I tried to convey with ", as long as the other requirements are met."